MUXSDKStats/MUXSDKStats/MUXSDKPlayerBindingManager.m

```diff
@@ -0,0 +1,68 @@
+//
+//  MUXSDKPlayerBindingManager.m
+//  MUXSDKStats
+//
+//  Created by Nidhi Kulkarni on 1/30/20.
+//  Copyright © 2020 Mux, Inc. All rights reserved.
+//
+
+#import "MUXSDKPlayerBindingManager.h"
+
+@interface MUXSDKPlayerBindingManager()
+
+// Player bindings for which we have dispatched the viewinit, customer player & video data, & playerready events to Core
+@property (nonatomic, retain) NSMutableSet *playerReadyBindings;
+
+@end
+@implementation MUXSDKPlayerBindingManager
+
+- (id)init {
+    self = [super init];
+    if (self) {
+        self.playerReadyBindings = [[NSMutableSet alloc] init];
+    }
+    return self;
+}
+
+
+- (BOOL) hasInitializedPlayerBinding:(NSString *) name {
+    return [self.playerReadyBindings containsObject:name];
+}
+
+- (void) initializeCoreForPlayer:(NSString *_Nonnull) name {
+    if (![self hasInitializedPlayerBinding: name]) {
+        MUXSDKPlayerBinding *binding = [self.viewControllers valueForKey:name];
```

Comment:
well this name confused me for a minute, but then I realized it's just been this way from the beginning, and it's not necessarily a ViewController

Reply:
Yeah, it confused me as well, we should rename this. I'll file a ticket